demos/theme/index.html

```diff
@@ -457,11 +457,113 @@ <h2 class="mdc-typography--headline demo-component-section__heading">
+              <input class="mdc-radio__native-control" type="radio" id="demo-radio-4" name="demo-radio-group-2">
+              <div class="mdc-radio__background">
+                <div class="mdc-radio__outer-circle"></div>
+                <div class="mdc-radio__inner-circle"></div>
+              </div>
+            </div>
+            <label id="demo-radio-4-label" for="demo-radio-4">Custom 2</label>
+          </div>
+        </figure>
+
+        <figure class="demo-radio-group">
+          <div class="mdc-form-field demo-radio-form-field">
+            <div class="mdc-radio demo-radio--custom">
+              <input class="mdc-radio__native-control" type="radio" id="demo-radio-5" checked name="demo-radio-group-3">
+              <div class="mdc-radio__background">
+                <div class="mdc-radio__outer-circle"></div>
+                <div class="mdc-radio__inner-circle"></div>
+              </div>
+            </div>
+            <label id="demo-radio-5-label" for="demo-radio-5">Custom 3</label>
+          </div>
+          <div class="mdc-form-field demo-radio-form-field">
+            <div class="mdc-radio demo-radio--custom">
+              <input class="mdc-radio__native-control" type="radio" id="demo-radio-6" name="demo-radio-group-3">
+              <div class="mdc-radio__background">
+                <div class="mdc-radio__outer-circle"></div>
+                <div class="mdc-radio__inner-circle"></div>
+              </div>
+            </div>
+            <label id="demo-radio-6-label" for="demo-radio-6">Custom 4</label>
+          </div>
+        </figure>
+
+        <figure class="demo-radio-group">
+          <div class="mdc-form-field demo-radio-form-field">
+            <div class="mdc-radio mdc-radio--disabled">
+              <input class="mdc-radio__native-control" type="radio" id="demo-radio-7" disabled checked name="demo-radio-group-4">
+              <div class="mdc-radio__background">
+                <div class="mdc-radio__outer-circle"></div>
+                <div class="mdc-radio__inner-circle"></div>
+              </div>
+            </div>
+            <label id="demo-radio-7-label" for="demo-radio-7">Disabled 1</label>
+          </div>
+          <div class="mdc-form-field demo-radio-form-field">
+            <div class="mdc-radio mdc-radio--disabled">
+              <input class="mdc-radio__native-control" type="radio" id="demo-radio-8" disabled name="demo-radio-group-4">
+              <div class="mdc-radio__background">
+                <div class="mdc-radio__outer-circle"></div>
+                <div class="mdc-radio__inner-circle"></div>
+              </div>
+            </div>
+            <label id="demo-radio-8-label" for="demo-radio-8">Disabled 2</label>
+          </div>
+        </figure>
+      </section>
     </main>
 
     <script src="/assets/material-components-web.js" charset="utf-8"></script>
     <script>
       (function() {
+        'use strict';
```

Comment:
What's the reason for adding this? (I see it in a few demos apparently but don't know why)

Reply:
I noticed it in some other demo pages and figured it wouldn't hurt to add it here. It's mainly just a "please make sure I don't do anything monumentally and preventably stupid" switch 😄